src/zap/zapimport.cpp

```diff
@@ -69,7 +68,7 @@ ZapImportTable::ModuleReferenceEntry * ZapImportTable::GetModuleReference(CORINF
     pEntry->m_wAssemblyRid = (USHORT) assemblyIndex;
```

Comment:
`ZapImportTable` should be deleted too.  We should not need it at all.

Reply:
If you're OK with these changes, I can merge them and submit the `ZapImportTable` refactor in a subsequent PR.

Reply:
I do not think the change is an improvement in the current state. It is really just renaming `CORCOMPILE_IMPORT_TABLE_ENTRY` to `USHORT`.

Reply:
Reasonable. I'll close this and re-open once I have something more substantial.